lib/apis/v3/felixconfig.go

```diff
@@ -356,6 +356,11 @@ type FelixConfigurationSpec struct {
 	// Set source-destination-check on AWS EC2 instances. Accepted value must be one of "DoNothing", "Enabled" or "Disabled".
 	// [Default: DoNothing]
 	AWSSrcDstCheck *AWSSrcDstCheckOption `json:"awsSrcDstCheck,omitempty" validate:"omitempty,oneof=DoNothing Enable Disable"`
+
+	// Prevent routing loops to service IPs that are not in use, by dropping or rejecting
```

Comment:
Maybe add `When service IP advertisement is enabled, ...` otherwise users might be confused that this only works in that case?